tests/e2e/test_internal_trades.py

```diff
@@ -166,6 +165,54 @@ def test_zero_buffer_trade(self):
         internal_trades = InternalTokenTransfer.internal_trades(internal_transfers)
         self.assertEqual(len(internal_trades), 0 * 2)
 
+    def test_buffer_trade_with_missing_price_from_pricesUSD(self):
+        """
+        tx: 0x80ae1c6a5224da60a1bf188f2101bd154e29ef71d54d136bfd1f6cc529f9d7ef
+        CRVCX is not part of the priceUSD list from dune, still we are finding the internal trade
+        """
+        internal_transfers = get_internal_transfers(
+            dune=self.dune_connection,
+            tx_hash="0x80ae1c6a5224da60a1bf188f2101bd154e29ef71d54d136bfd1f6cc529f9d7ef",
+            period_start=self.period_start,
+            period_end=self.period_end,
+        )
+        internal_trades = InternalTokenTransfer.internal_trades(internal_transfers)
+
+        self.assertEqual(len(internal_trades), 1 * 2)
+        self.assertEqual(
+            token_slippage(
+                "0xD533a949740bb3306d119CC777fa900bA034cd52", internal_transfers
+            ),
+            0,
+        )
+        self.assertEqual(
+            token_slippage(
+                "0x62B9c7356A2Dc64a1969e19C23e4f579F9810Aa7", internal_transfers
+            ),
+            0,
+        )
```

Comment:
So we check the slippage on both tokens. Are we hoping they are being excluded? I see the description mentions one token. CRVCX...